\documentclass{article}


% Bibliography
\usepackage{natbib}
\bibpunct{(}{)}{;}{a}{}{;}

% Use 'It was found that A is B (Name 1234)' style
\setcitestyle{authoryear,open={},close={}}

% Affiliations
\usepackage{authblk}
\title{
  Transmembrane helices are also 
  an overlooked source of major histocompatibility complex class II epitopes
  and evolutionary more conserved than expected by chance
}
\author[1]{Richèl J.C. Bilderbeek}
\author[1]{Maxim Baranov}
\author[1]{Geert van den Bogaart}
\author[1]{Frans Bianchi}
\affil[1]{GBB, University of 
Groningen, Groningen, The Netherlands}

% Use double spacing
\usepackage{setspace}
\doublespacing

\usepackage{listings}
\usepackage{hyperref}
\usepackage{todonotes}
\usepackage{verbatim}
\usepackage{pgf}
\usepackage{bm}
\usepackage{multirow}
\usepackage{amsfonts}
\usepackage{array}
\usepackage{array}
\usepackage{booktabs}
\newcolumntype{C}[1]{>{\centering\arraybackslash}p{#1}}
\newcolumntype{L}[1]{>{\raggedright\arraybackslash}p{#1}}
\usepackage{longtable}

\usepackage{tkz-graph}
\usetikzlibrary{arrows,automata}
\usetikzlibrary{calc}
\usetikzlibrary{arrows.meta}

% sidewaysfigure
\usepackage{rotating}

% Style of listings
% From http://r.789695.n4.nabble.com/
%   How-to-nicely-display-R-code-with-the-LaTeX-package-listings-tp4648110.html
\usepackage{fancyvrb} 
\definecolor{codegreen}{rgb}{0,0.6,0}
\definecolor{codegray}{rgb}{0.5,0.5,0.5}
\definecolor{codepurple}{rgb}{0.58,0,0.82}
\definecolor{backcolor}{rgb}{0.95,0.95,0.92}
\lstdefinestyle{mystyle}{
  language=R,% set programming language
  basicstyle=\ttfamily\small,% basic font style
  commentstyle=\color{gray},% comment style
  numberstyle=\scriptsize,% use small line numbers
  numbersep=10pt,% space between line numbers and code
  tabsize=2,% sizes of tabs
  showstringspaces=false,
  captionpos=b,% positioning of the caption below
  breaklines=true,% automatic line breaking
  escapeinside={(*}{*)},% escaping to LaTeX
  fancyvrb=true,% verbatim code is typset by listings
  extendedchars=false,% prohibit extended chars (chars of codes 128--255)
  alsoletter={.<-},% becomes a letter
  alsoother={$},% becomes other
  otherkeywords={!=, ~, $, \&, \%/\%, \%*\%, \%\%, <-, <<-, /},
  deletekeywords={c}% remove keywords 
}
\lstset{style=mystyle}

% Adds numbered lines. Not yet...
% \usepackage{lineno}
% \linenumbers

%comments
\newcommand{\frans}[1]{\textcolor{blue}{\textbf{[FB: #1]}}}
\newcommand{\richel}[1]{\textcolor{orange}{\textbf{[RB: #1]}}}

\begin{document}

\maketitle

\begin{abstract}

Transmembrane helices (TMHs) in the human proteome
are an overrepresented potential source of epitopes on major 
histocompatibility complex (MHC) class I for the majority of HLA-I haplotypes. 
It is unknown if this same overrepresentation
also holds for the proteome of a pathogen.
Additionally, it is unknown if TMH-II is also likelier to present
TMHs than expected by chance. 
Lastly, if the immune system if more tailored to detect polypeptides
derived from TMHs, can we confirm this by detecting the evolutionary
conservation of this behavior?
This study shows that MHC-I [also has/does not have] more
epitopes derived from a TMH for a pathogen proteome, when compared with
a host proteome.
Additionally, MHC-II binds to polypeptides derived from TMHs 
[less/equally/more] often than expected by chance.
Lastly, this study shows that in the mutation rate in TMHs of a pathogen
is [lower/as much/higher], hinting at 
evolutionary [conservation/neutrality/disruptive selection].
Our findings suggest that the immune system is [less/neutral/more]
vigilant to TMHs than expected by chance and this has left [a/no]
signal in the evolutionary history of the pathogen.

\end{abstract}

{\bf Keywords:} antigen presentation, membrane proteins, bioinformatics, 
adaptive immunity, transmembrane domain, epitopes, T lymphocyte, 
MHC-1, MHC-I, MHC-2, MHC-II, COVID-19

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\paragraph{Immune response}

\richel{Check against stupidity}
Our immune system fights invaders on a daily basis.
These invaders can be fungi, bacteria or viruses.
The innate immune response is its first general 
and immediate strategy, where the acquired immune response
needs time to develop its specialized and more effective
combat forces.

\paragraph{Immune response by MHC-I}

\richel{Check against stupidity}
All nucleated cells in humans present randomly sampled polypeptides
fragments to the surroundings of the cell using MHC class I molecules.
If a cell gets infected by a virus, also the virus' polypeptides
will be presented at the cell surface. The viral antigen is detected 
by cytotoxic T lymphocytes, which will kill the infected cell.

\paragraph{Immune response by MHC-II}

\richel{Check against stupidity}
Bacteria and fungi do not infect a cell with their foreign DNA,
but are detected by the proteins on their cell walls. MHC class II molecules 
bind to specific polypeptide fragments. This binding allows T and B cells,
which all express MHC-II, to detect a pathogen and trigger the response to
kill the pathogen. Note that the foreign proteins of a viral envelope 
can be detected in this way as well, which is commonly used in vaccination.

\paragraph{Non-random epitopes}

\richel{Check against stupidity}
Any human's immune system detects only a selection of all possible
polypeptide fragments. Each MHC molecule can only express (MHC-I)
or detect (MHC-II) a couple of polypeptides. An increased detection
is obtained by expressing a wide variety of
MHCs, which is assured by the human leukocyte antigen gene complex (HLA).
However, a different HLA will result in a different variety of MHCs,
that will display different polypeptide fragments. Additionally, there
may have been selection the HLA to display and or detect different
polypeptide fragments.

\paragraph{MHC-I}

\richel{Check against stupidity}
For MHC-I, it was found that predicted epitopes derived 
from human transmembrane helices (TMHs)
are over-presented by all HLA-A and 
most HLA-B super types (\cite{bianchi2017}). 
One explanation is that the presentation of TMHs 
may have an evolutionary advantage for 
the (human) host, as TMHs have a reduced variability 
due to the functional requirement of being able to span a lipid 
bilayer. 
Due to this, pathogens have a lower chance to develop an escape mutation,
as many mutations will result in a disfunctional TMH.
Note that the mechanism by which a cell presents its TMHs is
yet unknown.

\paragraph{MHC-II}

\richel{Check against stupidity}
If presentation of TMHs on MHC-I would bring an evolutionary advantage 
in the recognition of pathogens by the immune system, 
it would follow that this is equally important for MHC-II, 
especially as CD4+ T cell help is needed for a long lasting CD8+ T cell 
response \cite{novy2007cd4}. 
The mechanism to detect foreign TMHs by MHC-II would be unknown, 
similar to the discovery that MHC-I presents TMHs.

\paragraph{Evolutionary arms race}

\richel{Check against stupidity}
The human immune system and human pathogen are in an evolutionary
arms race, in which our immune systems is selected for the detection
of pathogens, whereas pathogens are selected to avoid being detected.
If the human immune system is more prone to display (and possibly 
detect) polypeptides derived from TMHs, one may expect to find 
an evolutionary signal in pathogens as well. 

\paragraph{Evolutionary signal of unknown strength for virus}

\richel{Check against stupidity}
Regarding human viruses, we do not know the extent to which the human
immune systems selects on these, nor which loci are selected
upon \richel{so I should read the literature on that}. 
We predict that there is a strong 
evolutionary conservation in regions that are essential for viral proteins'
functions. Because these regions are rare in a viral proteome,
and differ widely between pathogens, we expect that the human immune system
is not tailored to detect these sites. TMHs, however, have less 
functional constraints yet are more general. Because TMHs are more general,
we expect it for the immune system to pay off to be more attentive towards
TMHs. 
Additionally, viruses are not only selected for their evasion of an 
immune response, yet are selected to have a high reproductive value.
Because the strength of the evolutionary signal is unknown,
we use a strong data set to be able to detect it.

\paragraph{Evolutionary signal of unknown strength for bacteria}

\richel{Check against stupidity}
Regarding human pathogenic bacteria, we do not know the extent to which the 
human immune systems selects on these, nor which loci are selected
upon \richel{so I should read the literature on that}.
Because most bacterial pathogens are generalists, that is,
can infect multiple hosts, we expect the selection pressure exerted
by the human immune response to be weak.
Also, it is unknown if bacterial TMHs are detected by MHC-II at all.
Due to this, the strength of the evolutionary signal is unknown,
thus we need a strong data set to be able to detect it.

\paragraph{Data sets}

\richel{Check against stupidity}
To detect an evolutionary signal for selection on the TMHs of viruses, 
we use the many time-dated COVID-19 proteomes.
We can assume that only after the moment that COVID-19 spilled over 
to humans, it has mostly been the human immune system selecting against
its detection, although this is only part of the selection for 
having a high transmission rate.
To detect an evolutionary signal for selection on the TMHs of bacteria, 
we use the proteomes of Mycobacterium. Mycobacterium is \richel{something}.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Hypotheses}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{itemize}
  \item $\mathcal{H}_1$: MHC-I has the same percentage of epitopes overlapping
    with TMHs in Homo sapiens as in COVID-19 as in Mycobacterium
  \item $\mathcal{H}_2$: MHC-II has the same percentage of epitopes overlapping
    with TMHs as expected by chance in Homo sapiens and COVID-19 and Mycobacterium
  \item $\mathcal{H}_3$: The TMHs of both COVID-19 and Mycobacterium
    have the same mutation rate 
    as the cytosolic and extracellular parts of their transmembrane proteins
\end{itemize}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Methods}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{MHC-I}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

We used the same analysis as \cite{bianchi2017},
except that instead of a human reference proteome,
we use the proteome of the first sequenced COVID-19 strain (\cite{wu2020new},
GenBank ID of MN908947.3, \url{https://www.ncbi.nlm.nih.gov/nuccore/MN908947})
and a reference Mycobacterium tuberculosis 
proteome (\url{https://www.ebi.ac.uk/reference_proteomes}, UP000001584, 
83332 MYCTU).

Bianchi and colleagues obtained a distribution of 
percentages of MHC-I epitopes overlap with TMHs in Homo sapiens
for the different HLA haplotypes, with an average of 5.3\%.
We obtained a similar distribution of percentages of MHC-I epitopes that 
overlap with TMHs for the different HLA haplotypes, but then applied to
COVID-19 and Mycobacterium.

We compare the distributions of humans and each pathogen
using a two-sample 
Kolmogorov-Smirnov (KS) test. The KS test determines if two samples
are derived from the same distribution, without making assumptions
regarding the shape of that distribution. 

We will reject
the null hypothesis that MHC-I has the same percentage of epitopes 
overlapping with TMHs in Homo sapiens compared to each pathogen when 
the KS statistic $D_{n,m}$ follows the relationship as shown in 
equation \ref{eq:ks}, for a significance level $\alpha = 0.05$
and $n = m = 13$ HLA haplotypes.

\begin{equation}
   D_{n,m} > \frac{1}{\sqrt{n}} \cdot \sqrt{ -\ln(\frac{\alpha}{2}) \cdot \frac{1 + \frac{n}{m}}{2} }
\  \label{eq:ks}
\end{equation}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{MHC-2}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

For MHC-II, we do the same analysis as MHC-I, except for
MHC-II haplotypes.



	% Transmembrane helices and strong MHC-II-binding peptides
	% were predicted for a tuberculosis reference proteome 
	% using the \verb;epitopeome; R package (\cite{epitopeome}).
	% We picked the three MHC-II alleles that are most abundant 
	% in the current human population, 
	% which are DRB4*0101, HLA-DPA10103-DPB10402 
	% \richel{
	%   We decided for DPB1*0402, but there are multiple alleles matching.
	%   I just picked the first one of the alleles matching, which are:
	%   HLA-DPA10x-DPB10402, where x equals 103-110, 201-204, 301-303 or 401
	% }
	% and DQA1*0501/DQB1*0301 \richel{reference here}
	% \frans{
	%   Neem aan dat dit die ref is naar dat stukje van most common haplotype in human population. Misschien moeten we even de coverage erbij melden. 
	%   Greenbaum, Jason, et al. "Functional classification of class II human leukocyte antigen (HLA) molecules reveals seven different supertypes and a surprising degree of repertoire sharing across supertypes." Immunogenetics 63.6 (2011): 325-335.
	% }.
	% The 5\% peptides with the lowest IC50 values were defined as binders.
	% We then simply counted the number of amino acids that were present inside the
	% cell, within the membrane or outside of the cell, as well as if it was part 
	% of a strong MHC-II binding site, as shown in figure \ref{tab:results}.


	% The \verb;epitopeome; R package (\cite{epitopeome}) binds 
	% together the \verb;tmhmm; (\cite{tmhmm}) and \verb;netmhc2pan; 
	% (\cite{netmhc2pan}) 
	% R packages. \verb;tmhmm; provides an R interface to
	% TMHMM (\cite{krogh2001predicting, sonnhammer1998hidden}), a tool
	% to predict where membrane proteins' amino acids are located within the
	% membrane.   
	% \verb;netmhc2pan; provides an R interface to
	% NetMHC2pan (\cite{jensen2018improved}), a tool
	% to predict MHC-II binding to proteins.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Evolutionary conservation}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

	% \frans{
	%   Ik heb zitten denken over de evolutie naast conservering is occurrence ook heel belangrijk. Moeten we de mate van conservering niet wegen ten opzichte van de occurrence.
	%   Wat ik daarmee bedoel is data als je kiest voor een mega geconseveerd epitope maar van dat type komen er maar 40 in een proteome voor versus een minder geconserveerd epitope maar daar zijn wel 100.000 varianten van dan heb je meer kans om 1 van die 100.000 te pakken toch? 
	% }
	% The evolutionary conservation of TMHs was measured from a
	% DNA alignment of multiple transmembrane proteins in multiple
	% species of the mycobacterium bacterial family
	% \richel{how obtained exactly? how alignment done?}
	% \frans{
	% Via entrez en blast 
	%   Ik he been transcript van Maxim ik zal dit proberen te vertalen naar een stukje tekst. 
	% }
	% .
	% Using the \verb;tmhprot; R package (\cite{tmhprot}), the DNA alignment is split
	% into two alignments, one for the TMH parts, another for the non-TMH
	% parts. Each alignment was tested by the \verb;mcbette; 
	% R package (\cite{mcbette}) to select the Bayesian inference model with
	% the highest evidence (a.k.a. the marginal likelihood) using the nested
	% sampling approach as described in \cite{maturana2018model},
	% using the popular Bayesian phylogenetic tool 
	% BEAST2 (\cite{bouckaert2014beast}) in the back-end.
	% \verb;mcbette; used a set of 40 candidate inference models, 
	% consisting of all combinations of 
	% 4 site models (JC, HKY, TN, GTR), 
	% 2 clock models (strict, RLN) and 
	% 5 tree priors (Yule, BD, CBS, CCP, CEP).

	% For each alignment, the inference model with the highest evidence
	% is used in a Bayesian inference. From each Bayesian posterior,
	% the parameter estimates regarding mutation rates (including clock rate)
	% were obtained and compared to determine which realized mutation rate is lower.
	% The Markov chain Monte Carlo was set up in such a way that the effective sample
	% size for the likelihood of the inference model is above the recommended value
	% of 200 (\cite{bouckaert2014beast}).

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{MHC-I}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Figure \ref{fig:1} shows the percentages of epitopes overlapping 
with TMHs for a human, COVID-19 and Mycobacterium proteome.
The KS test to determine if these percentages are sampled from
a same distribution resulted in a p-value of \richel{unknown},
which makes us reject/accept the hypothesis that these percentages
are sampled from the same distribution. 

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{bbbq_1/fig_bbbq_1_watermarked.png}
  \caption{
    \% epitopes overlapping with transmembrane helix
    for a human, COVID-19 and Mycobacterium proteome.
    \richel{
      the data underlying this figure has been simulated
    }
  }
  \label{fig:1}
\end{figure}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{MHC-II}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{bbbq_2/fig_bbbq_2_watermarked.png}
  \caption{
    \% epitopes overlapping with transmembrane helix
    for a human, COVID-19 and Mycobacterium proteome.
    \richel{
      the data underlying this figure has been simulated
    }
  }
  \label{fig:2}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Evolutionary conservation}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

	% Table \ref{tab:results} shows the location and binding strength for the
	% tuberculosis proteome.

	% \begin{figure}[ht]
	%   \includegraphics[width=\textwidth]{bbbq_3/figure_1.png}
	%   \label{fig:1}
	% \end{figure}

	% \begin{figure}[ht]
	%   \includegraphics[width=\textwidth]{figure_1_5.png}
	%   \label{fig:1_5}
	% \end{figure}

	% \begin{figure}[ht]
	%   \includegraphics[width=\textwidth]{figure_3.png}
	%   \label{fig:3}
	% \end{figure}




	% \input{table_imoimo.latex}
	% % has label tab:results
	% \richel{
	%   I'd enjoy 
	%   (1) a row with 'expected by chance', 
	%   (2) using percentages instead,
	%   (3) merging inside and outside
	% }

	% The inference model with the highest evidence in the
	% TMH-only alignment was [yet unknown] and [also yet unknown]
	% for the non-TMH alignment. Individual model weights are shown
	% in tables \ref{tab:evidences_tmh} 
	% and \ref{tab:evidences_non_tmh}.

	% The Bayesian inference resulted in [a distribution of mutation rates],
	% as shown in [absent figure].
	% The ESSes of the Bayesian parameter estimates was above 200, exact values
	% are shown in tables \ref{tab:esses_tmh} and \ref{tab:esses_non_tmh}.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Conclusion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

We found that the percentages of epitopes overlapping 
with TMHs for a human and COVID-19 proteome are 
[similar/different]. In other words, the
epitopes that MHC-I presents are [as/not as] likely 
to be derived from TMH within either a human host and its viral pathogen.

We found that the percentages of epitopes overlapping 
with TMHs for a human and Mycobacterium proteome are 
[similar/different]. In other words, the
epitopes that MHC-I presents are [as/not as] likely 
to be derived from TMH within either a human host and its bacterial pathogen.

	% We conclude that MHC-II binds to TMH peptides with a higher/lower/equal
	% probability than expected by chance. 

	% We conclude that the evolutionary conservation if the TMH parts of membrane
	% proteins is higher/less/equal compare to its non-TMH counterparts.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

We concluded that the
epitopes that MHC-I presents are [as/not as] likely 
to be derived from TMH within either a human host and its viral pathogen.
Because the full COVID-19 only has 4 TMHs \richel{check}, the percentages
of MHC-I epitopes being part of a TMH are likelier to be affected by
stochasticity. We chose to use COVID-19 regardless, as the thousands
of its time-dated genomic sequences are ideal for determining the 
evolutionary conservation of MHC-I detecting TMHs. 

We concluded that the
epitopes that MHC-I presents are [as/not as] likely 
to be derived from TMH within either a human host and its bacterial pathogen.
Because a bacterium does not infect a cell, thus its polypeptides
will not be presented by MHC-I, this result is [unexpected/expected]

	% \frans{
	%   De TMH evolutie data van COVID is missachien interresanter. Ik zie het als volgt. 
	%   COVID is een mooi voorbeeld van een virus 
	%   Myco van een bacterie
	%   In de bacterie is duidelijk dat tenmisnte voor MHCII TMHs zijn overgrepresenteerd virus niet. 
	%   Dus verhaal kan zijn dat TMHs belangrijk voot bacteirele afweer niet zozeer voor virussen. 
	% 
	%   Hiervoor moeten we MHC-I doen voor myco
	%   Kijken wat evolutie data doet voor virus dan evt kijken wat dit doet voor myco en vergelijken. 
	% }
	% 
	% We compared the mutation rates between the TMH and non-TMH part of
	% multiple mycobacterium species. Where we expect no variation 
	% in mutation rate for every TMH amino acid,
	% \richel{
	%   we can test this, but unsure if that would make sense
	% }
	% we know that non-TMH part will have regions of different evolutionary
	% conservation: functional domains, especially in protein-protein
	% interactions will be strongly conserved, due to an even more constrained
	% set of peptides that enable a certain function.

	% \richel{
	%   Note that most bacteria are opportunistic pathogens.
	%   Note that most bacteria are generalists.
	%   Note that most bacteria have different cell membranes (and walls), that
	%   may have different functional constraints than a human cell membrane
	% }

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Acknowledgments}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

We thanks to Geert van den Bogaart for his wisdom.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Data Accessibility}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

All code is archived at \url{http://github.com/richelbilderbeek/someplace},
with DOI \url{https://doi.org/12.3456/zenodo.1234567}.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Authors' contributions}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

RJCB and FB conceived the idea for this research. 
RJCB wrote the code.
RJCB and FB wrote the article.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% MEE style
\bibliographystyle{mee}
\bibliography{article}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\appendix
\section{Supplementary materials}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{MHC-I}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{table}
  \input{bbbq_1/bbbq_1_percentages.latex}
  \caption{
    Percentage of MHC-I epitopes overlapping with transmembrane helix.
    \richel{This is simulated data}
  }
  \label{table:bbbq_1_percentages}
\end{table}

\begin{table}
  \input{bbbq_1/bbbq_1_stats_covid.latex}
  \caption{
    Kolmogorov-Smirnov test results
    \richel{Done on the simulated data}
  }
  \label{table:bbbq_1_stats_covid}
\end{table}

\begin{table}
  \input{bbbq_1/bbbq_1_stats_myco.latex}
  \caption{
    Kolmogorov-Smirnov test results
    \richel{Done on the simulated data}
  }
  \label{table:bbbq_1_stats_myco}
\end{table}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{MHC-II}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{table}
  \input{bbbq_2/bbbq_2_percentages.latex}
  \caption{
    Percentage of MHC-I epitopes overlapping with transmembrane helix.
    \richel{This is simulated data}
  }
  \label{table:bbbq_2_percentages}
\end{table}

\begin{table}
  \input{bbbq_2/bbbq_2_stats_covid.latex}
  \caption{
    Kolmogorov-Smirnov test results
    \richel{Done on the simulated data}
  }
  \label{table:bbbq_2_stats_covid}
\end{table}

\begin{table}
  \input{bbbq_2/bbbq_2_stats_myco.latex}
  \caption{
    Kolmogorov-Smirnov test results
    \richel{Done on the simulated data}
  }
  \label{table:bbbq_2_stats_myco}
\end{table}

\end{document}
