\documentclass{article}

% Bibliography
\usepackage{natbib}
\bibpunct{(}{)}{;}{a}{}{;}

% Use 'It was found that A is B (Name 1234)' style
\setcitestyle{authoryear,open={},close={}}

% Affiliations
\usepackage{authblk}
\title{
  Transmembrane helices are also 
  an overlooked source of major histocompatibility complex class II epitopes
  and evolutionary more conserved than expected by change
}
\author[1]{Rich\`el J.C. Bilderbeek}
\author[1]{?Rampal S. Etienne}
\author[2]{Frans Bianchi}
\affil[1]{Groningen Institute for Evolutionary Life Sciences, University of Groningen, Groningen, The Netherlands}
\affil[2]{Frans' Institute, University of Groningen, Groningen, The Netherlands}

% Use double spacing
\usepackage{setspace}
\doublespacing

\usepackage{listings}
\usepackage{hyperref}
\usepackage{todonotes}
\usepackage{verbatim}
\usepackage{pgf}
\usepackage{bm}
\usepackage{multirow}
\usepackage{amsfonts}
\usepackage{array}
\usepackage{array}
\usepackage{booktabs}
\newcolumntype{C}[1]{>{\centering\arraybackslash}p{#1}}
\newcolumntype{L}[1]{>{\raggedright\arraybackslash}p{#1}}
\usepackage{longtable}

\usepackage{tkz-graph}
\usetikzlibrary{arrows,automata}
\usetikzlibrary{calc}
\usetikzlibrary{arrows.meta}

% sidewaysfigure
\usepackage{rotating}

% Style of listings
% From http://r.789695.n4.nabble.com/How-to-nicely-display-R-code-with-the-LaTeX-package-listings-tp4648110.html
\usepackage{fancyvrb} 
\definecolor{codegreen}{rgb}{0,0.6,0}
\definecolor{codegray}{rgb}{0.5,0.5,0.5}
\definecolor{codepurple}{rgb}{0.58,0,0.82}
\definecolor{backcolor}{rgb}{0.95,0.95,0.92}
\lstdefinestyle{mystyle}{
  language=R,% set programming language
  basicstyle=\ttfamily\small,% basic font style
  commentstyle=\color{gray},% comment style
  % numbers=left,% display line numbers on the left side
  numberstyle=\scriptsize,% use small line numbers
  numbersep=10pt,% space between line numbers and code
  tabsize=2,% sizes of tabs
  showstringspaces=false,% do not replace spaces in strings by a certain character
  captionpos=b,% positioning of the caption below
  breaklines=true,% automatic line breaking
  escapeinside={(*}{*)},% escaping to LaTeX
  fancyvrb=true,% verbatim code is typset by listings
  extendedchars=false,% prohibit extended chars (chars of codes 128--255)
  alsoletter={.<-},% becomes a letter
  alsoother={$},% becomes other
  otherkeywords={!=, ~, $, \&, \%/\%, \%*\%, \%\%, <-, <<-, /},% other keywords
  deletekeywords={c}% remove keywords 
}
\lstset{style=mystyle}

% Adds numbered lines
\usepackage{lineno}
\linenumbers

% Rename 'Abstract' to 'Summary 
\usepackage[english]{babel}
\addto{\captionsenglish}{\renewcommand{\abstractname}{Summary}}

%comments
\newcommand{\frans}[1]{\textcolor{blue}{\textbf{[FB: #1]}}}
\newcommand{\richel}[1]{\textcolor{orange}{\textbf{[RB: #1]}}}
\newcommand{\rampal}[1]{\textcolor{green}{\textbf{[RSE: #1]}}}


\begin{document}

\maketitle

\begin{abstract}

Transmembrane helices (TMHs) are an overlooked source of 
major histocompatibility complex (MHC) class I epitopes.
It is unknown why this is the case and 
whether MHC-II also is an overlooked source of epitopes.
This study investigates if there is an evolutionary conservation 
of a pathogen's TMHs and if this is matches 
with MHC-II's over/equally/under-presenting TMHs.
We find that ...
\end{abstract}

{\bf Keywords:} antigen presentation, membrane proteins, bioinformatics, 
adaptive immunity, transmembrane domain, epitopes, T lymphocyte, MHC-2

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

For MHC-1, we know that epitopes derived from transmembrane helices (TMHs) are 
over-presented by all human leukocyte antigen (HLA)-A and most HLA-B super 
types [\cite{bianchi2017transmembrane}].

A hypothesis offered by that same paper is that the presentation of 
especially TMHs may have an evolutionary advantage for the (human) host.
The argumentation given is that the presentation of parts of TMHs
is likelier to detect pathogenic proteins, as TMHs are already conserved
due to the functional requirement of being able to span a membrane.
Additionally, there are fewer escape mutations possible for the pathogen,
as relatively less of these will result in a functional TMH.

However, that argumentation can also go the other way: as there are less
polypeptide fragments from (the less amount of) possible TMHs, 
it is likelier that host and pathogen
share the same protein sequence, 
resulting the pathogen avoiding detection.

Within the immune response, MHC-I is unique in [doing something].
The other most important participant in the immune response in MHC-II,
which is more involved in detecting bacterial pathogens, 
e.g. tuberculosis, [and others].

From the earlier research done on MHC-I, it naturally follows to check
if MHC-II are likelier to present TMHs than expected by chance.

Extending the previous study on MHC-I, we also measure the evolutionary
conservation of TMH epitopes in Mycoplasma pathogens.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Methods}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Transmembrane helices and strong MHC-II-binding peptides
were predicted for a tuberculosis reference proteome 
from \url{https://www.ebi.ac.uk/reference_proteomes}
using the \verb;epitopeome; 
\richel{
  I'd enjoy a better name
}
R package [\cite{epitopeome}].
We picked the three MHC-II alleles that are most abundant 
in the current human population, 
which are DRB4*0101, HLA-DPA10103-DPB10402 
\richel{
  We decided for DPB1*0402, but there are multiple alleles matching.
  I just picked the first one of the alleles matching, which are:
  HLA-DPA10x-DPB10402, where x equals 103-110, 201-204, 301-303 or 401
}
and DQA1*0501/DQB1*0301 \richel{reference here}.
The 5\% peptides with the lowest IC50 values were defined as binders.
We then simply counted the number of amino acids that were present inside the
cell, within the membrane or outside of the cell, as well as if it was part 
of a strong MHC-II binding site, as shown in figue \ref{tab:results}.


The \verb;epitopeome; R package \cite{epitopeome} binds together 
the \verb;tmhmm; [\cite{tmhmm}] and \verb;netmhc2pan; [\cite{netmhc2pan}] 
R packages. \verb;tmhmm; provides an R interface to
TMHMM [\cite{krogh2001predicting}, \cite{sonnhammer1998hidden}], a tool
to predict where membrane proteins' amino acids are located within the
membrance.   
\verb;netmhc2pan; provides an R interface to
NetMHC2pan [\cite{jensen2018improved}], a tool
to predict MHC-II binding to proteins.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\input{table_imoimo.latex}
% has label tab:results
\richel{
  I'd enjoy 
  (1) a row with 'expected by chance', 
  (2) using percentages instead,
  (3) merging inside and outside
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Conclusion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Acknowledgements}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Thanks to Geert van den Bogaart for his help \richel{reword}.
We thank the Netherlands 
Organization for Scientific Research (NWO) for financial support 
through a VICI grant awarded to RSE.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Data Accessibility}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

All code is archived at \url{http://github.com/richelbilderbeek/someplace},
with DOI \url{https://doi.org/12.3456/zenodo.1234567}.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Authors' contributions}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

RJCB, FB and RSE conceived the idea for this research. 
RJCB wrote the code.
FB supplied the DNA alignment.
RJCB, FB and RSE contributed to writing the article.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% MEE style
\bibliographystyle{mee}
\bibliography{article}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\appendix

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Appendix section 1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\end{document}
