%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% This is just an example/guide for you to refer to when submitting manuscripts to Frontiers, it is not mandatory to use Frontiers .cls files nor frontiers.tex  %
% This will only generate the Manuscript, the final article will be typeset by Frontiers after acceptance.   
%                                              %
%                                                                                                                                                         %
% When submitting your files, remember to upload this *tex file, the pdf generated with it, the *bib file (if bibliography is not within the *tex) and all the figures.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% Version 3.4 Generated 2018/06/15 %%%
%%% You will need to have the following packages installed: datetime, fmtcount, etoolbox, fcprefix, which are normally inlcuded in WinEdt. %%%
%%% In http://www.ctan.org/ you can find the packages and how to install them, if necessary. %%%
%%%  NB logo1.jpg is required in the path in order to correctly compile front page header %%%

\documentclass[utf8]{frontiersSCNS} % for Science, Engineering and Humanities and Social Sciences articles
%\documentclass[utf8]{frontiersHLTH} % for Health articles
%\documentclass[utf8]{frontiersFPHY} % for Physics and Applied Mathematics and Statistics articles

%\setcitestyle{square} % for Physics and Applied Mathematics and Statistics articles
\usepackage{url,hyperref,lineno,microtype,subcaption}
\usepackage[onehalfspacing]{setspace}

\linenumbers


% Leave a blank line between paragraphs instead of using \\


\def\keyFont{\fontsize{8}{11}\helveticabold }
\def\firstAuthorLast{Sample {et~al.}} %use et al only if is more than 1 author
\def\Authors{Rich√®l J.C. Bilderbeek\,$^{1}$, Maksim V. Baranov\,$^{1}$, Geert van den Bogaart\,$^{1}$ and Frans Bianchi\,$^{1,*}$}


% Affiliations should be keyed to the author's name with superscript numbers and be listed as follows: Laboratory, Institute, Department, Organization, City, State abbreviation (USA, Canada, Australia), and Country (without detailed address information such as city zip codes or street names).
% If one of the authors has a change of address, list the new address below the correspondence details using a superscript symbol and use the same symbol to indicate the author in the author list.
\def\Address{$^{1}$Molecular Immunology, GBB, University of Groningen, Groningen, The Netherlands}
% Provide the exact contact address (this time including street name and city zip code) and email of the corresponding author
\def\corrAuthor{Frans Bianchi, Nijenborgh 7, 9747 AG Groningen}

\def\corrEmail{f.bianchi@rug.nl}




\begin{document}
\onecolumn
\firstpage{1}

\title[Transmembrane helices are 
  an overlooked and evolutionarily conserved
  source of major histocompatibility complex class I and II epitopes
]{Transmembrane helices are 
  an overlooked and evolutionarily conserved
  source of major histocompatibility complex class I and II epitopes
} 

\author[\firstAuthorLast ]{\Authors} %This field will be automatically populated
\address{} %This field will be automatically populated
\correspondance{} %This field will be automatically populated

\extraAuth{}% If there are more than 1 corresponding author, comment this line and uncomment the next one.
%\extraAuth{corresponding Author2 \\ Laboratory X2, Institute X2, Department X2, Organization X2, Street X2, City X2 , State XX2 (only USA, Canada and Australia), Zip Code2, X2 Country X2, email2@uni2.edu}


\maketitle


\begin{abstract}

%%% Leave the Abstract empty if your article does not require one, please see the Summary Table for full details.
\section{}
Cytolytic T cell responses are predicted to be biased towards membrane proteins. 
The peptide-binding grooves of most alleles
of histocompatibility complex class I (MHC-I) are relatively hydrophobic, 
therefore peptide fragments derived from human transmembrane helices (TMHs) are predicted to be presented more often as would be expected 
based on their abundance in the proteome. However, the physiological reason of why membrane proteins might be over-presented is unclear. 
In this study, we show that the predicted over-presentation of TMH-derived peptides is general, as it is predicted for bacteria and viruses 
and for both MHC-I and MHC-II, and confirmed by re-analysis of epitope databases. Moreover, we show that TMHs are evolutionarily more conserved, 
because single nucleotide polymorphisms (SNPs) are present 
relatively less frequently in TMH-coding chromosomal regions 
compared to regions coding for extracellular and cytoplasmic protein regions. 
Thus, our findings suggest that 
both cytolytic and helper T cells are more tuned to respond to membrane proteins, 
because these are evolutionary more conserved. 
We speculate that TMHs are less prone to mutations 
that enable pathogens to evade T cell responses.


\tiny
 \keyFont{ \section{Keywords:} antigen presentation, membrane proteins, bioinformatics, adaptive immunity, transmembrane domain, transmembrane helix, epitopes, T lymphocyte, MHC-I, MHC-II, evolutionary conservation}
\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% For Original Research Articles \citep{conference}, Clinical Trial Articles \citep{article}, and Technology Reports \citep{patent}, the introduction should be succinct, with no subheadings \citep{book}. For Case Reports the Introduction should include symptoms at presentation \citep{chapter}, physical exams and lab results \citep{dataset}.

% \paragraph{Immune response}

Our immune system fights diseases and infections from pathogens, 
such as fungi, bacteria or viruses. 
An important part of the acquired immune response, 
that develops specialized and more specific recognition of pathogens 
than the innate immune response, 
are T cells which recognize peptides, called epitopes, derived from antigenic proteins presented on Major Histocompatibility Complexes (MHC) class I and II on the cell surface. 

% \paragraph{Classification of HLA}

The MHC proteins are heterodimeric complexes encoded by the
HLA (Human Leukocyte Antigens) genes.
In humans, the peptide binding groove of MHC-I is made by only the alpha subunit. There are three classical alleles of MHC-I, hallmarked by a highly polymorphic alpha chain called HLA-A, HLA-B and HLA-C, that all present epitopes to cytolytic T cells. 
For MHC-II, both the alpha and the beta chains contribute to the peptide binding groove. There are three classical alleles of MHC-II as well, called HLA-DR, HLA-DQ and HLA-DP, that all present epitopes to helper T cells.
Each MHC complex can present a subset of all possible peptides.
For example, HLA-A and HLA-B have no overlap in which
epitopes they bind \cite{lund2004definition}.
Moreover, the HLA genes of humans are highly polymorphic, with hundreds 
to thousands of different alleles, 
and each different allele presents a different subset of peptides \cite{marsh2010nomenclature}.

% \paragraph{HLAs increase detection range}

Humans express a limited set of MHC alleles
and therefore an individual's immune system detects 
only a fraction of all possible peptide fragments. 
However, at the population level, the coverage of pathogenic peptides that are detected 
is very high, because of the highly polymorphic MHC genes.
It is therefore believed that MHC polymorphism improves immunity at the population level, 
as mutations in a protein that disrupt a particular MHC presentation at the individual level, 
so-called escape mutations, 
will not affect MHC presentation for all alleles present in the population \cite{sommer2005importance}.

% \paragraph{Epitope prediction}

Many studies are aimed at identifying the repertoire of epitopes that are presented 
in any of the different alleles to determine which epitopes will result in an immune response, 
as this will for instance aid the design of vaccines. 
These studies have led to the development of prediction algorithms 
that allow for very reliable \emph{in silico} 
predictions of the peptide binding affinities
\cite{larsen2010identification,schellens2008unanticipated,tang2011genome}.
For example, S. Tang et al. \cite{tang2011genome} found that, 
of the 432 peptides that were predicted to bind to an MHC allele,
86\% were experimentally confirmed to do so. 

% \paragraph{TMHs}

Using these prediction algorithms, 
we recently showed that peptides derived 
from transmembrane helices (TMHs) 
are likely to be more frequently presented by MHC-I 
than expected based on their abundance \cite{bianchi2017},
which is in line with a previous study 
by Istrail et al \cite{istrail2004comparative},
demonstrating that N-terminal signal sequences 
are likely to be presented within major histocompatibility complexes, 
due their hydrophobic nature. 
Moreover, we showed that some well-known immunodominant peptides stem from TMHs. 
This over-presentation is attributed to the fact 
that the peptide-binding groove of most MHC-I alleles
is relatively hydrophobic, 
and therefore hydrophobic TMH-derived peptides have a higher affinity 
to bind than their soluble hydrophobic 
counterparts. 

TMHs are hydrophobic 
as they need to span the hydrophobic lipid bilayer of cellular membranes.
They consist of an alpha helix of, on average, 23 amino acids in length. 
TMHs can also be predicted with high accuracy from a protein sequence 
by bioinformatics approaches \cite{krogh2001predicting,kall2004combined,arai2004conpred,jones2007improving,klammer2009metatm,wang2019efficient}. 
For example, a study by Jones \cite{jones2007improving} found that,
from 184 transmembrane proteins (TMPs) with known topology, 
80\% of the TMH predictions of these proteins
matched the experimental findings.
TMHs are common structures in the proteins of humans and microbes. 
Different TMH prediction tools estimate that 15-39\% of all proteins 
in the human proteome contain at least one TMH \cite{ahram2006estimation}.
However, the physiological reason why peptides derived from TMHs 
would be presented more often than peptides 
stemming from soluble (i.e., extracellular or cytoplasmic) protein regions is unknown. 
In this study, we hypothesized that the presentation of 
TMH residues is evolutionarily preferred, 
since TMHs are less prone to undergo escape mutations. 
One reason to expect such a reduced 
variability (and hence evolutionary conservation) in TMHs, 
is that these are restricted in their variability 
by the functional requirement to span a lipid bilayer. 
This limits 
many of the amino acids present in TMHs 
to have hydrophobic side chains 
\cite{hessa2007molecular,jones1994model}.
Therefore, we speculated that the TMHs of pathogens 
might have a lower chance to develop escape mutations, 
as that will result in a dysfunctional TMH 
and render the protein inactive.

This study had two objectives. 
First, we aimed to generalize our findings by predicting
the antigenic presentation
from different kingdoms of life in both MHC-I and -II. 
From these \emph{in silico} predictions, we conclude that TMH-derived
epitopes from a human, viral and bacterial proteome
are likely to be presented more often than expected by chance 
for most alleles of MHC-I and II. 
We confirmed the presentation of TMH-derived peptides 
by re-analysis of peptides from The Immune Epitope Database (IEDB) \cite{vita2019immune}.
Second, we tested our hypothesis that TMHs 
are more evolutionary conserved than soluble protein regions.
Our analysis of human single nucleotide polymorphisms (SNPs) showed 
that random point mutations are indeed less likely
to occur within TMHs. 
These findings strengthen the emerging notion 
that TMHs are important for the T cell-mediated adaptive immune system, 
and hence are of  
overlooked importance in vaccine development.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Article types}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

For requirements for a specific article type please refer to the Article Types on any Frontiers journal page. Please also refer to  \href{http://home.frontiersin.org/about/author-guidelines#Sections}{Author Guidelines} for further information on how to organize your manuscript in the required sections or their equivalents for your field

% For Original Research articles, please note that the Material and Methods section can be placed in any of the following ways: before Results, before Discussion or after Discussion.

\section{Manuscript Formatting}

\subsection{Heading Levels}

%There are 5 heading levels

\subsection{Level 2}
\subsubsection{Level 3}
\paragraph{Level 4}
\subparagraph{Level 5}

\subsection{Equations}
Equations should be inserted in editable format from the equation editor.

\begin{equation}
\sum x+ y =Z\label{eq:01}
\end{equation}

\subsection{Figures}
Frontiers requires figures to be submitted individually, in the same order as they are referred to in the manuscript. Figures will then be automatically embedded at the bottom of the submitted manuscript. Kindly ensure that each table and figure is mentioned in the text and in numerical order. Figures must be of sufficient resolution for publication \href{http://home.frontiersin.org/about/author-guidelines#ResolutionRequirements}{see here for examples and minimum requirements}. Figures which are not according to the guidelines will cause substantial delay during the production process. Please see \href{http://home.frontiersin.org/about/author-guidelines#GeneralStyleGuidelinesforFigures}{here} for full figure guidelines. Cite figures with subfigures as figure \ref{fig:2}B.


\subsubsection{Permission to Reuse and Copyright}
Figures, tables, and images will be published under a Creative Commons CC-BY licence and permission must be obtained for use of copyrighted material from other sources (including re-published/adapted/modified/partial figures and images from the internet). It is the responsibility of the authors to acquire the licenses, to follow any citation instructions requested by third-party rights holders, and cover any supplementary charges.
%%Figures, tables, and images will be published under a Creative Commons CC-BY licence and permission must be obtained for use of copyrighted material from other sources (including re-published/adapted/modified/partial figures and images from the internet). It is the responsibility of the authors to acquire the licenses, to follow any citation instructions requested by third-party rights holders, and cover any supplementary charges.

\subsection{Tables}
Tables should be inserted at the end of the manuscript. Please build your table directly in LaTeX.Tables provided as jpeg/tiff files will not be accepted. Please note that very large tables (covering several pages) cannot be included in the final PDF for reasons of space. These tables will be published as \href{http://home.frontiersin.org/about/author-guidelines#SupplementaryMaterial}{Supplementary Material} on the online article page at the time of acceptance. The author will be notified during the typesetting of the final article if this is the case. 

\section{Nomenclature}

\subsection{Resource Identification Initiative}
To take part in the Resource Identification Initiative, please use the corresponding catalog number and RRID in your current manuscript. For more information about the project and for steps on how to search for an RRID, please click \href{http://www.frontiersin.org/files/pdf/letter_to_author.pdf}{here}.

\subsection{Life Science Identifiers}
Life Science Identifiers (LSIDs) for ZOOBANK registered names or nomenclatural acts should be listed in the manuscript before the keywords. For more information on LSIDs please see \href{http://www.frontiersin.org/about/AuthorGuidelines#InclusionofZoologicalNomenclature}{Inclusion of Zoological Nomenclature} section of the guidelines.


\section{Additional Requirements}

For additional requirements for specific article types and further information please refer to \href{http://www.frontiersin.org/about/AuthorGuidelines#AdditionalRequirements}{Author Guidelines}.

\section*{Conflict of Interest Statement}
%All financial, commercial or other relationships that might be perceived by the academic community as representing a potential conflict of interest must be disclosed. If no such relationship exists, authors will be asked to confirm the following statement: 

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

\section*{Author Contributions}

The Author Contributions section is mandatory for all articles, including articles by sole authors. If an appropriate statement is not provided on submission, a standard one will be inserted during the production process. The Author Contributions statement must describe the contributions of individual authors referred to by their initials and, in doing so, all authors agree to be accountable for the content of the work. Please see  \href{http://home.frontiersin.org/about/author-guidelines#AuthorandContributors}{here} for full authorship criteria.

\section*{Funding}
Details of all funding sources should be provided, including grant numbers if applicable. Please ensure to add all necessary funding information, as after publication this is no longer possible.

\section*{Acknowledgments}
This is a short text to acknowledge the contributions of specific colleagues, institutions, or agencies that aided the efforts of the authors.

\section*{Supplemental Data}
 \href{http://home.frontiersin.org/about/author-guidelines#SupplementaryMaterial}{Supplementary Material} should be uploaded separately on submission, if there are Supplementary Figures, please include the caption in the same file as the figure. LaTeX Supplementary Material templates can be found in the Frontiers LaTeX folder.

\section*{Data Availability Statement}
The datasets [GENERATED/ANALYZED] for this study can be found in the [NAME OF REPOSITORY] [LINK].
% Please see the availability of data guidelines for more information, at https://www.frontiersin.org/about/author-guidelines#AvailabilityofData

\bibliographystyle{frontiersinSCNS_ENG_HUMS} % for Science, Engineering and Humanities and Social Sciences articles, for Humanities and Social Sciences articles please include page numbers in the in-text citations
%\bibliographystyle{frontiersinHLTH&FPHY} % for Health, Physics and Mathematics articles
\bibliography{bbbq_article}

%%% Make sure to upload the bib file along with the tex file and PDF
%%% Please see the test.bib file for some examples of references

\section*{Figure captions}

%%% Please be aware that for original research articles we only permit a combined number of 15 figures and tables, one figure with multiple subfigures will count as only one figure.
%%% Use this if adding the figures directly in the mansucript, if so, please remember to also upload the files when submitting your article
%%% There is no need for adding the file termination, as long as you indicate where the file is saved. In the examples below the files (logo1.eps and logos.eps) are in the Frontiers LaTeX folder
%%% If using *.tif files convert them to .jpg or .png
%%%  NB logo1.eps is required in the path in order to correctly compile front page header %%%

\begin{figure}[h!]
\begin{center}
\includegraphics[width=10cm]{logo1}% This is a *.eps file
\end{center}
\caption{ Enter the caption for your figure here.  Repeat as  necessary for each of your figures}\label{fig:1}
\end{figure}


\begin{figure}[h!]
\begin{center}
\includegraphics[width=15cm]{logos}
\end{center}
\caption{This is a figure with sub figures, \textbf{(A)} is one logo, \textbf{(B)} is a different logo.}\label{fig:2}
\end{figure}

%%% If you are submitting a figure with subfigures please combine these into one image file with part labels integrated.
%%% If you don't add the figures in the LaTeX files, please upload them when submitting the article.
%%% Frontiers will add the figures at the end of the provisional pdf automatically
%%% The use of LaTeX coding to draw Diagrams/Figures/Structures should be avoided. They should be external callouts including graphics.

\end{document}
