---
title: "COVID-19 dataset strength"
author: Richel J.C. Bilderbeek
output: pdf_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r, message=FALSE}
library(testthat)
library(pureseqtmr)
```

Must be installed:

```{r}
expect_true(is_pureseqtm_installed())
```


```{r cache=TRUE}
fasta_filename <- "UP000464024.fasta"
expect_true(file.exists(fasta_filename))
topology <- pureseqtmr::predict_proteome_topology(fasta_filename)
```

```{r}
n_non_tmh_aas <- sum(stringr::str_count(topology$topology, "0"))
message("number of non-TMH amino acids, n_non_tmh_aas: ", n_non_tmh_aas)
```

```{r}
n_tmh_aas <- sum(stringr::str_count(topology$topology, "1"))
message("number of TMH amino acids, n_tmh_aas: ", n_tmh_aas)
```

```{r}
n_aas <- n_non_tmh_aas + n_tmh_aas
proteome_size <- n_aas
message("number of amino acids, n_aas: ", n_aas)
message("proteome size, proteome_size: ", proteome_size)
```

```{r}
perc_aas <- n_tmh_aas / (n_tmh_aas + n_non_tmh_aas)
message("percentage of proteome that is TMH, perc_aas: ", perc_aas)
```

```{r}
n_tmhs <- sum(tally_tmhs(topology)$n_tmhs) 
message("number of TMHs, n_tmhs: ", n_tmhs)
```

Get the proteome mutation rate, by converting it from the 
known DNA mutation rate:

```{r}
genome_size <- 30000 # nucleotides

dna_evolution_rate_per_genome_per_year <- 26 # From GISAID
# (Same as bar2020sars, that esimates a DNA 
# evolution rate per nucleotide per year at 0.001)
```

```{r}
rel_size_proteome_to_genome <- 
  proteome_size / genome_size
message("rel_size_proteome_to_genome: ", rel_size_proteome_to_genome)
```


\begin{itemize}
	\item Proteome size: 9701 (from reference proteome)
	\item Size of proteome compared to genome: $\frac{9701}{3 \cdot 10^4} \approx \frac{1}{3}$ \richel{no/little non-coding DNA indeed}
	\item Fraction of mutations that change AA: $\frac{20}{64}$ 
	\item Evolution rate: $\approx 0.00092$ per amino acid per year (as $\frac{1}{3} \cdot \frac{20}{64} \cdot 0.00088$)
	\item TMHs per proteome: 26 (from table \ref{table:covid_tmhs})
	\item AAs per TMHs: 23 (from Bianchi et al.)
	\item AAs per proteome that is TMH: $21 \cdot 23 = 483$
	\item Mutations in TMH per proteome year: $\approx 0.444$ (as $483 \cdot 0.00092$)
	\item Number of sequenced proteomes: 5600 (GISAID, genomes)
	\item Pandemic duration: $\approx \frac{1}{3}$ year
	\item Expected number of observed TMH mutations: $\approx 800$ (as $0.444 \cdot 5600 \cdot \frac{1}{3}$)
	\item Chance that a mutation may lose/acquire epitope binding: $\frac{1}{100}$ \richel{Just a guess, based on weak selection}
	\item Expected number of observed TMH mutations that cause a transition: $\approx 8$ (as $800 \cdot \frac{1}{100}$)
\end{itemize}
